src/ImageSharp/PixelFormats/PixelImplementations/PixelOperations/A8.PixelOperations.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using SixLabors.ImageSharp.Formats;
+
+namespace SixLabors.ImageSharp.PixelFormats
+{
+    /// <content>
+    /// Provides optimized overrides for bulk operations.
+    /// </content>
+    public partial struct A8
+    {
+        /// <summary>
+        /// Provides optimized overrides for bulk operations.
+        /// </summary>
+        internal class PixelOperations : PixelOperations<A8>
+        {
+            /// <inheritdoc />
+            public override PixelTypeInfo GetPixelTypeInfo() => PixelTypeInfo.Create<A8>(PixelAlphaRepresentation.Unassociated);
```

Comment:
Note for later:
we may want to cache the value into a field/lazy property so we can query it efficiently including it's potential usage in `To/FromVector4`.

Reply:
Done.